features/other_b/series_lock.feature

```diff
@@ -4,84 +4,60 @@ Feature: Locked and partially locked series
+      And I set up the draft "Antivalentine" as part of a series "Antiholidays"
       And I check "work_restricted"
-    And I check "series-options-show"
-    And I fill in "work_series_attributes_title" with "Antiholidays"
-    And I press "Preview"
-  Then I should see "Draft was successfully created."
-    And I should see "Part 1 of the Antiholidays series"
-  When I press "Post"
-    And I go to fandomer's series page
-  Then I should see "Antiholidays"
-  When I am logged out
-   And I go to fandomer's series page
-  Then I should not see "Antiholidays"
-  When I am logged in as "reccer" with password "password"
-    And I go to fandomer's series page
-  Then I should see "Antiholidays"
-  When I view the series "Antiholidays"
-  Then I should see "Humbug"
-  When I am logged out
-    And I am logged in as "fandomer" with password "password"
-    And I go to the new work page
-    And I select "Not Rated" from "Rating"
-    And I check "No Archive Warnings Apply"
-    And I fill in "Fandoms" with "Supernatural"
-    And I fill in "Work Title" with "Antivalentine"
-    And I fill in "content" with "The not-love-story of how they met and how they got into trouble"
-    And I check "work_restricted"
-    And I check "series-options-show"
-    And I select "Antiholidays" from "work_series_attributes_id"
-    And I press "Preview"
-  Then I should see "Draft was successfully created."
-    And I should see "Part 2 of the Antiholidays series"
-  When I view the series "Antiholidays"
-  Then I should see "Works: 1"
-    And I should not see "Antivalentine"
-  When I view the work "Humbug"
-  Then I should not see "»" within "dd"
-  When I edit the work "Antivalentine"
-    And I uncheck "work_restricted"
-    And I press "Preview"
-  Then I should see "Part 2 of the Antiholidays series"
-  When I press "Post"
-  Then I should see "Part 2 of the Antiholidays series"
-    And I should not see the image "title" text "Restricted"
-    And I should see "«" within "dd.series"
-  When I am logged out
-    And I view the series "Antiholidays"
-  Then I should see "Antivalentine"
-    But I should not see "Humbug"
-  When I view the work "Antivalentine"
-  Then I should see "Part 1 of the Antiholidays series"
-    And I should not see "»" within "dd"
-  When I am logged in as "reccer" with password "password"
-    And I go to fandomer's series page
-  Then I should see "Antiholidays"
-  When I view the series "Antiholidays"
-  Then I should see "Works: 2"
-    And I should see "Humbug"
-    And I should see "Antivalentine"
+      And I press "Preview"
+    Then I should see "Draft was successfully created."
+      And I should see "Part 2 of the Antiholidays series"
+    When I view the series "Antiholidays"
+    Then I should see "Works: 1"
+      And I should not see "Antivalentine"
+    When I view the work "Humbug"
+    Then I should not see "»" within "dd"
+    When I edit the work "Antivalentine"
+      And I uncheck "work_restricted"
```

Comment:
This could be `And I unlock the work` (it's the same thing, but if we ever change the id of the checkbox, it makes it so there are fewer places we need to update the tests).